### DISCUSSION TITLE
No salt cell information on the dashboard

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#621] [Q&A]
I am just getting started with the poolController.  I have a Pentair Easytouch 2 4 controller with an Intellichlor salt cell.  I do not have any salt cell information available on the dashboard.  Can you please tell me what I need to do in order to get salt cell info on the dashboard?

Thank you for your help.

**Comment by rstrouse** [comment]
Try this.
1. Go into the dashPanel configuration and delete the IntelliChlor.
2. Add the IntelliChlor back in but please Select EasyTouch as the Master.

Chances are you declared the IntelliChlor as an Externally controlled device or you set it up for Nixie control.  If you would like me to verify this post up a replay.
https://github.com/tagyoureit/nodejs-poolController/wiki/How-to-capture-all-packets-for-issue-resolution

**Reply by FlaMike** [reply]
Please forgive my ignorance...  Should I edit a .json file in the dashPanel directory or is this something I can edit on the dashPanel UI?  I don't see anything that relates to the Intellichlor in either location.

**Reply by FlaMike** [reply]
![image](https://user-images.githubusercontent.com/41341956/192155878-b8c4b3da-5e59-4f79-810c-a440b80f9246.png)

**Reply by FlaMike** [reply]
Here is a replay.  Thanks again for all of your help!   I really prefer not to be such a bother.
[replay.zip](https://github.com/tagyoureit/nodejs-poolController/files/9641338/replay.zip)

**Reply by rstrouse** [reply]
Oh the chlorinator isn't even enabled in the ET.  Simply click the gears at the top right of the screen.  Then click on the Chemistry tab.  From there click on add Controller.  

On this screen select EasyTouch 2 4 as the master and click on the button.  It will then show up on the chemistry bubble.  With ET you may need to refresh the screen.  You can also add it from the EasyTouch controller by enabling the IntelliChlor.
![image](https://user-images.githubusercontent.com/47839015/192157906-44f81ec7-3bec-47a3-8b8b-b0582a3819eb.png)

**Reply by FlaMike** [reply]
OMG.  EZ PZ, I just didn't understand why the chlorinator wasn't there and discovered along with the rest of the equipment, let alone how to add it back (it was running from the Easytouch panel all along).  I also didn't realize there is a "button" above the drop down menu.  I had tried earlier to add the chlorinator from this screen but didn't understand how to save the setting of ET (only recognized the cancel button & the go away box as options in the window).

Thanks again for much for your help!!!  I hope to leave you alone for some time other than to say how much I like your system :-)

**Reply by FlaMike** [reply]
The finished product, integrated into Home Assistant!  Thanks again for your patience, help, support, & great software!

![image](https://user-images.githubusercontent.com/41341956/192164281-26bd2143-a0bc-40c0-8267-2c942ce81497.png)
